docs/firstobject.md

```diff
@@ -76,78 +76,81 @@ There is of course a lot more to Nodes, Content Types, Formatters, Widgets and i
 
-## That first Digital Object
+![Add Content](../imgs/04_ingest_add-content_digital-object.png)
 
 ### My Metadata
 
-What you see here is a `Form Mode` in action, with the Title field exposed as a simple text input (`Field Widget`), a Multi Step Web Form that will ingest metadata into a field of type Strawberry Field (where all the magic happens) 
-attached to that field using a `Webform Field Widget`, an editorial/advanced Block on the right side and a Workflow/state drop down (`Save as`) at the bottom. 
+What you see below is a `Form Mode` in action, with the Title field exposed as a simple text input (`Field Widget`), a Multi Step Web Form that will ingest metadata into a field of type Strawberry Field (where all the magic happens) attached to that field using a `Webform Field Widget`, an editorial/advanced Block on the right side and a Workflow/state drop down (`Save as`) at the bottom.
+
+Let's fill out the form to begin our ingest. We recommend to use similar values as the ones shown in the screen capture to make following the tutorial easier.
+
+All fields with a red `*` are required fields. Be sure to enter a `Title`, `Media Type`, `Description`, and `Location, Place or Address`. When you are finished, press `Move on to next step` at the bottom of the form.
+
+![Ingest Step 1 - MyMetadata](../imgs/01_ingest_step-one_MyMetadata.png)
+
+**Tip:** When entering a location, place or address you will need to click on the `Search OpenStreet Maps` button, which is what that big red arrow is pointing to in the screenshot above.
 
-![Ingest Step1](../imgs/MyMetadata_1.jpg)
+**Note:** For now, let's leave the state of this Digital Object as `Draft`.
 
-Fill the form out. We recommend to use similar values as the ones shown in the screen capture to make following the tutorial easier. 
 
-<details><summary>Why does this look different than Repository X?</summary>
+<details><summary>"Why does this look different than Repository X?"</summary>
 <div>
 
-We assume you come from a world where repositories define different Content types and the shape, the fields and values (Schema) are fixed and set by someone else or at least quite complicated to configure. This is where Archipelago differs and starts to propose its own style. You noticed that there is a single Content Type named `Digital Object` and you have here a single Web Form. So how does this allows you to have Images, sequences, Videos, audio, 3D images, etc? 
+We assume you come from a world where repositories define different Content types and the shape, fields and values (Schema) are fixed and set (or at least quite complicated to configure) by someone else. This is where Archipelago differs and starts to propose its own style. You may have noticed that there is a single Content Type named `Digital Object` and you have here a single Web Form. So how does this allow you to have images, sequences, videos, audio, 3D images, etc?
```

Comment:
> (or at least quite complicated to configure)

Not sure about this... can you expand? What we are trying to say here is that Content Models are defined/set by the system. Not the qualitative aspect of its easy/complex to do.E.g some systems don't even allow users to set content types at all